topics/transcriptomics/tutorials/scrna-scanpy-pbmc3k/slides.html

```diff
@@ -0,0 +1,169 @@
+objectives:
+  - "Learn some background knowledge about microbiome"
+  - "Understand metatranscriptomics"
+  - "Familiarize yourself with KEGG terminology"
+  - "Understand how network(pathway) analysis works"
+time_estimation: 1H
+key_points:
+  - "Analyzing differential expression for (meta)transcriptomics data"
+  - "Validating the p-value distribution of the differential expression analysis"
+  - "Finding the most differentially expressed subnetwork in a gene functional network using Heinz"
+  - "Interpreting the pinpointed subnetwork"
+contributors:
+  - mtekman
+
+---
+
+# Single Cell RNA Pre-processing
+
+- Barcoding and (de-)multiplexing
+- Quantification
+
+
+---
+
+# Barcoding Cells
+
+.center[![fig1](../../images/scrna_pbb_barcodes_add.svg)]
+
+---
+
+# Single Cell RNA Downstream Analysis
+
+- Filtering
+- Normalising
+  - Confounder Removal
+- Dimension Reduction
+- Clustering
+- Annotation
+
+---
+
+# Filtering: Cell and Gene
+
+.center[![fig2](../../images/raceid_libsize.svg)]
+
+---
+
+# Normalisation: Technical Variation
+
+.center[![fig3](../../images/raceid_technical_variation.svg)]
+
+---
+
+# Normalisation: Biological Variation
+
+
+.center[![fig4](../../images/raceid_cellcycle.svg)]
+
+---
+
+# DimRed: Relatedness of Cells
+
+.center[![fig5](../../images/scrna_knn.svg)]
+
+
+???
+Build a KNN graph from distance matrix:
+  * If P and q share distance which is 
+
+---
+
+# DimRed: Projection
+
+.center[![fig6](../../images/raceid_dimred.svg)]
+
+- Can use tSNE, PCA, UMAP
+
+---
+
+### Community Clustering: Louvain
+
+.center[![fig7](../../images/commgraph1.svg)]
+
+Aim: Maximise internal links and minimise external links
+
+---
+
+# Community Clustering: Louvain
+
+.center[![fig8](../../images/commgraph2.svg)]
+
+* Randomly pick a cell and try to place it in a neighbour's cluster
+  * Accept if *Internal:External* increases
+  * Reject and pick another
+
+---
+
+# Cell Type: Identifying Cluster Types
+
+.pull-left[![fig9](../../images/singlecellplot3.png)]
+.pull-right[![fig10](../../images/singlecellplot4.png)]
+
+---
+
+# Clustering: Hard vs Soft
+
+.pull-left[
+![fig11](../../images/singlecellplot3.png)
+
+*Hard*
+  * Big spaces between clusters
+  * Cell types are well defined and the clustering reflects that
+
+]
+
+--
+
+.pull-right[
+![fig12](../../images/10xdata.png)
+
+*Soft*
+  * Clusters bleed into one another
+  * Cell types seem to intermingle with one another.
+
+]
+
+???
+Why? Why would there be clusters so close to one another?
+
+---
+
+# Continuous Phenotypes:
+
+.center[![fig13](../../images/raceid_contpheno.svg)]
+
+* Cells aren't discrete, they transition
+* Continuously changing over time from a less mature type to more mature type
+
+---
+
+## Interactive Environments: live.usegalaxy.eu
+
+
+<video style="width:100%;height:auto" controls>
+    <source src="https://zenodo.org/record/3687100/files/live_cellxgene.mp4" />
+</video>
+
+
+---
+
+## Test
```

Comment:
is that needed?

Reply:
No, I just could not get the slides the build for a long time so was testing various ways.